modules/openapi-generator/src/main/resources/csharp-netcore/Configuration.mustache

```diff
@@ -10,6 +10,10 @@ using System.IO;
 using System.Linq;
 using System.Security.Cryptography.X509Certificates;
 using System.Text;
+using System.Net;
+using Polly;
+using Polly.Retry;
```

Comment:
This configuration object is meant to be framework agnostic.

By adding RestSharp and Polly here, this breaks our contract of allowing users to build without RestSharp as a dependency and now adds Polly into the mix.

If the decision here is to tightly couple Polly with the RestSharp implementation of ApiClient, please modify only the ApiClient (preferably via constructor or thread-safe property injection).